codegen/rest.go

```diff
@@ -225,14 +225,14 @@ end
 {{- $varName := varName $parameter.Name $parameter.Type $parameter.Schema.Ref }}
 {{- $varName := $varName | pascalToSnake }}
 {{- $varComment := varComment $parameter.Name $parameter.Type $parameter.Schema.Ref $parameter.Items.Type }}
-{{- if and (eq $parameter.Name "body") $parameter.Schema.Ref }}
+{{- if and (eq $parameter.In "body") $parameter.Schema.Ref }}
 {{- bodyFunctionArgsDocs $parameter.Schema.Ref }}
 {{- end }}
-{{- if and (eq $parameter.Name "body") $parameter.Schema.Type }}
--- @param body ({{ $parameter.Schema.Type }}) {{ $parameter.Description }}
+{{- if and (eq $parameter.In "body") $parameter.Schema.Type }}
+-- @param body ({{ $parameter.Schema.Type }}) {{ $parameter.Description | stripNewlines }}
```

Comment:
Some descriptions have embedded newlines which ends up messing with the generated code